docs/docsite/rst/roadmap/COLLECTIONS_10.rst

```diff
@@ -0,0 +1,87 @@
+
+.. note::
+
+  In general, it is in the discretion of the release manager to delay a release by 1-2 days for reasons such as personal (schedule) problems, technical problems (CI/infrastructure breakdown), and so on.
+  However, in case two releases are planned for the same day, a release of the latest stable version takes precedence. This means that if a stable Ansible 10 release collides with a pre-release of Ansible 11, the latter will be delayed.
+  If a Ansible 10 release collides with a stable Ansible 11 release, including 11.0.0, the Ansible 10 release will be delayed.
+
+
+Planned major changes
+=====================
+
+- The netapp.aws collection will be removed as it is unmaintained (https://github.com/ansible-community/ansible-build-data/issues/223).
+- The gluster.gluster collection will be removed as it is unmaintained (https://github.com/ansible-community/ansible-build-data/issues/249).
+- The community.sap collection will be removed as it is deprecated (https://github.com/ansible-community/ansible-build-data/issues/262).
+- The netapp.azure collection will be removed as it is unmaintained (https://github.com/ansible-community/ansible-build-data/issues/272).
+- The netapp.elementsw collection will be removed as it is unmaintained (https://github.com/ansible-community/ansible-build-data/issues/276).
+- The netapp.um_info collection will be removed as it is unmaintained (https://github.com/ansible-community/ansible-build-data/issues/280).
+- The community.azure collection will be removed as it is deprecated (https://github.com/ansible-community/ansible-build-data/issues/283).
+- The hpe.nimble collection will be removed as it is unmaintained (https://github.com/ansible-community/ansible-build-data/issues/285).
+
+You can install removed collections manually with ``ansible-galaxy collection install <collection_name>``.
+
+
+Ansible minor releases
+=======================
+
+Ansible 10.x follows ansible-core-2.17.x releases, so releases will occur approximately every four weeks. If ansible-core delays a release for whatever reason, the next Ansible 10.x minor release will be delayed accordingly.
+
+Ansible 10.x minor releases may contain new features (including new collections) but not backwards incompatibilities. In practice, this means we will include new collection versions where either the patch or the minor version number has changed but not when the major number has changed. For example, if Ansible-10.0.0 ships with community.crypto 2.3.0, Ansible-10.1.0 could ship with community.crypto 2.4.0 but not community.crypto 3.0.0.
+
+
+.. note::
+
+    Minor and patch releases will stop when Ansible-11 is released. See the :ref:`Release and Maintenance Page <release_and_maintenance>` for more information.
+
```

Comment:
Can we add the same note from https://github.com/ansible/ansible-documentation/pull/1023/commits/02aada27f053b07074199d25df01b8b9c3e1c4e3? @samccann requested the change for the PR to extend the Ansible 9 lifecycle, but the same thing applies here.